include/mfmg/dealii/amge_host.templates.hpp

```diff
@@ -52,18 +52,18 @@ struct MatrixFreeAgglomerateOperator
       : _mesh_evaluator(mesh_evaluator), _dof_handler(dof_handler),
         _constraints(constraints)
   {
+    _mesh_evaluator.initialize_agglomerate(dof_handler);
```

Comment:
I was wondering how on earth it compiled with your turning the data member into a const reference but now I see you used the argument :)